flow/project.py

```diff
@@ -1971,27 +1985,35 @@ def _print_progress(x):
                                               ignore_errors=ignore_errors,
                                               cached_status=cached_status)
 
-        groups = []
+        singleton_groups = []
         for group in self._groups.values():
             if len(group.operations) == 1:
                 for name in group.operations:
                     if name == group.name:
-                        groups.append(group)
-
-        op_results = list(tqdm(
-            iterable=map(_get_group_status, groups),
-            desc="Collecting job status per operation.",
-            total=len(groups)))
+                        singleton_groups.append(group)
+
+        def _generate_results_with_tqdm(iterable, map=None, desc=None, len_itr=None):
```

Comment:
Having the default be `None` means that `map` always needs to be passed or a `TypeError` will happen right?

Reply:
Yes, this is true.
I will change the default to `map`